run-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+/app/bin/rake db:drop db:create spec:prepare
```

Comment:
This file scares me because if somebody accidentally runs it, their local database gets dropped. Also, why are you running the specs separately and dropping databases rather than:
```/app/bin/bundle exec rspec --pattern spec/**/*_spec.rb``` or ```./bin/rake``` ?